railties/lib/rails/application/finisher.rb

```diff
@@ -141,6 +141,7 @@ def self.complete(_state)
       initializer :add_internal_routes do |app|
         if Rails.env.development?
           app.routes.prepend do
+            get "/rails"                 => "rails/tools#index",     internal: true
```

Comment:
This is _probably_ fine, but in case we eventually think `/rails` is too common to override, we might go with `/rails/tools`

Reply:
Prefer to use just `/rails` as anything more complicated will make it harder for developers to discover.